server/backend/src/domain/user/service/userEmailService.js

```diff
@@ -0,0 +1,49 @@
+import dotenv from 'dotenv';
+import crypto from 'crypto';
+import AppError from '../../../global/utils/appError.js';
+import redisCofig from '../../../global/config/redisCofig.js';
+import User from '../models/user.js';
+import sendEmail from '../../../global/utils/sendEmail.js';
+
+dotenv.config({ path: './.env' });
+
+export default {
+  sendPasswordResetEmail: async (resetReq) => {
+    // 1) 비밀번호 변경하려는 유저가 존재하는지 확인.
+    const users = await User.findTargetUserByEmail(resetReq.getUserEmail);
+
+    if (users.length === 0) {
+      throw new AppError('Not Found', 404, 'E404AD');
+    }
+
+    // 2) 비밀번호 재설정을 위한 토큰 발급
+    const token = crypto.randomBytes(20).toString('hex');
+    const resetToken = crypto.createHash('sha256').update(token).digest('hex');
+
+    // 3) 생성한 토큰 정보를 데이터베이스에 저장.
+    await redisCofig.redisClient.set(
+      resetToken,
+      users[0].userEmail,
+      'EX',
+      60 * 3,
+      () => {
+        console.log('데이터 저장 완료 3분후 삭제');
+      },
```

Comment:
```suggestion
      () => {},
```
혹시, 불필요한 console.log라면 삭제해도 괜츈할 것 같습니다!